### ISSUE TITLE
Tests failed

### ISSUE THREAD

**Issue by drnic** [garfunkel/go-realestatecomau#1]
The challenges of parsing HTML from never stagnant websites never cease. Currently the library is failing its sanity check:

```
$ go test ./... -v
=== RUN   TestGetInfo
2016/08/02 10:04:51 No results found
exit status 1
FAIL    github.com/garfunkel/go-realestatecomau 0.424s
```

**Comment by garfunkel** [comment]
Fixed. The bastards also went ahead and very slightly changed how inspections are represented too, so that's now fixed as well. Thanks for bringing this to my attention!

**Comment by drnic** [comment]
xoxo

On Tue, Aug 2, 2016 at 8:22 PM +1000, "Simon Allen" notifications@github.com wrote:

Closed #1.
